types/g2/index.d.ts

```diff
@@ -0,0 +1,674 @@
+// Type definitions for g2 3.0.5
+// Project: https://github.com/antvis/g2
+// Definitions by: yixin https://github.com/Leannechn;
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = G2;
+export as namespace G2;
+
+declare namespace G2 {
+  function track(option: boolean): void;
+  const version: string;
+  const Animate: Animate;
+  const Util: Util;
+  const Shape: Shape;
+
+  class Global {
+    setTheme(option: 'default' | 'dark'): void;
+    version: string;
+    trackable: boolean;
+    animate: boolean;
+    snapArray: Array<number>;
```

Comment:
@Leannechn I got the same error as Travis CI complains, but different from the Error in your image. I think there are 2 issues needed to be solved.
1. checkout to HEAD in your repo https://github.com/Leannechn/DefinitelyTyped and run the test again. Hope you get the same error, which has nothing to do with react-reduc-epic, as from Travis CI.
2. the index.d.ts file from types/g2 doesn't conform to what DefinitelyTyped requires. Please make the linter happy before makeing a pull request.

I cloned your code from you repo, and played around for a while. Found some issues.
```
> definitely-typed@0.0.1 test /tmp/DefinitelyTyped
> node node_modules/types-publisher/bin/tester/test.js --run-from-definitely-typed "types/g2"

Clean data
Clean logs
Clean output
# Typing Publish Report Summary
Started at Mon, 26 Mar 2018 03:38:18 GMT
Found 4162 typings folders in /tmp/DefinitelyTyped/types
Error: At 21:16 : Expected /<([^>]+)> */
    at Object.parseHeaderOrFail (/tmp/DefinitelyTyped/node_modules/definitelytyped-header-parser/index.js:47:15)
    at /tmp/DefinitelyTyped/node_modules/types-publisher/src/lib/definition-parser.ts:93:3
    at Generator.next (<anonymous>)
    at fulfilled (/tmp/DefinitelyTyped/node_modules/types-publisher/bin/lib/definition-parser.js:4:58)
    at <anonymous>
Error: Parsing failed.
    at fail (/tmp/DefinitelyTyped/node_modules/types-publisher/src/util/util.ts:320:11)
    at ChildProcess.child.on (/tmp/DefinitelyTyped/node_modules/types-publisher/src/util/util.ts:308:6)
    at ChildProcess.emit (events.js:180:13)
    at finish (internal/child_process.js:768:14)
    at process._tickCallback (internal/process/next_tick.js:112:11)
npm ERR! Test failed.  See above for more details.
```
Take a look at the error log, the test expected `/<([^>]+)> */`. What it really wants is a valid type annotation.
So I ran the linter, `npm run lint g2`. It complained `Array<T>` should be `T[]`. I corrected it, and ran `npm test types/g2` again, the error of this line was gone.

Hope this helps.